tests/test_internals/test_movie_interface.py

```diff
@@ -66,7 +66,7 @@ def get_metadata(self):
         return metadata
 
     def test_movie_starting_times(self):
-        starting_times = [np.float(np.random.randint(200)) for i in range(len(self.movie_files))]
+        starting_times = [float(np.random.randint(200)) for i in range(len(self.movie_files))]
```

Comment:
This is better because it removes the warning, but a better solution yet would be to use the vectorized numpy version:

```python
starting_times = np.random.randint(200, size=(len(self.movie_files), ) dtype=float)
```

Reply:
I don't think the performance would matter in this case though because it's just small test cases, so this PR is fine.

Reply:
Yes, I thought about it as well.

I was thinking at revisiting these tests on detail at some point in the future as we have the issue:
#449